prime-angular-frontend/src/app/modules/adjudication/pages/enrollee-adjudicator-documents/enrollee-adjudicator-documents.component.spec.ts

```diff
@@ -7,15 +10,28 @@ import { HttpClientTestingModule } from '@angular/common/http/testing';
 import { APP_CONFIG, APP_DI_CONFIG } from 'app/app-config.module';
 
 import { EnrolleeAdjudicatorDocumentsComponent } from './enrollee-adjudicator-documents.component';
+import { AdjudicatorDocumentsComponent } from '@adjudication/shared/components/adjudicator-documents/adjudicator-documents.component';
 import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
 
 describe('EnrolleeAdjudicatorDocumentsComponent', () => {
   let component: EnrolleeAdjudicatorDocumentsComponent;
   let fixture: ComponentFixture<EnrolleeAdjudicatorDocumentsComponent>;
 
+  let spyOnCreateEnrolleeAdjudicationDocument;
+  let spyOnGetEnrolleeAdjudicationDocumentDownloadToken;
+  let spyOnGetDocuments;
+
+  let mockEnrolleeId;
+  let mockAdjudicatorId;
+  let mockGuidArray;
+  let mockDocumentId;
+
   beforeEach(waitForAsync(() => {
     TestBed.configureTestingModule({
-      declarations: [EnrolleeAdjudicatorDocumentsComponent],
+      declarations: [
+        EnrolleeAdjudicatorDocumentsComponent,
+        AdjudicatorDocumentsComponent
```

Comment:
seems there is an undefined property error `removeFiles` from the `AdjudicatorDocumentsComponent`, both in codeclimate test and `ng test` local test.
```
Chrome Headless 98.0.4758.102 (Linux x86_64) ERROR
  An error was thrown in afterAll
  TypeError: Cannot read properties of undefined (reading 'removeFiles')
      at SafeSubscriber._next (http://localhost:9876/_karma_webpack_/webpack:/src/app/modules/adjudication/pages/enrollee-adjudicator-documents/enrollee-adjudicator-documents.component.ts:56:4)
      at SafeSubscriber.__tryOrUnsub (http://localhost:9876/_karma_webpack_/webpack:/node_modules/rxjs/_esm2015/internal/Subscriber.js:183:1)
      at SafeSubscriber.next (http://localhost:9876/_karma_webpack_/webpack:/node_modules/rxjs/_esm2015/internal/Subscriber.js:122:1)
      at Subscriber._next (http://localhost:9876/_karma_webpack_/webpack:/node_modules/rxjs/_esm2015/internal/Subscriber.js:72:1)
      at Subscriber.next (http://localhost:9876/_karma_webpack_/webpack:/node_modules/rxjs/_esm2015/internal/Subscriber.js:49:1)
      at Object.complete (http://localhost:9876/_karma_webpack_/webpack:/node_modules/rxjs/_esm2015/internal/observable/forkJoin.js:50:1)
      at Object.wrappedComplete (http://localhost:9876/_karma_webpack_/webpack:/node_modules/rxjs/_esm2015/internal/Subscriber.js:166:1)
      at SafeSubscriber.__tryOrUnsub (http://localhost:9876/_karma_webpack_/webpack:/node_modules/rxjs/_esm2015/internal/Subscriber.js:183:1)
      at SafeSubscriber.complete (http://localhost:9876/_karma_webpack_/webpack:/node_modules/rxjs/_esm2015/internal/Subscriber.js:168:1)
      at Subscriber._complete (http://localhost:9876/_karma_webpack_/webpack:/node_modules/rxjs/_esm2015/internal/Subscriber.js:79:1)
Chrome Headless 98.0.4758.102 (Linux x86_64): Executed 543 of 605 ERROR (0 secs / 6.288 secs)
Chrome Headless 98.0.4758.102 (Linux x86_64) ERROR
  An error was thrown in afterAll
  TypeError: Cannot read properties of undefined (reading 'removeFiles')
      at SafeSubscriber._next (http://localhost:9876/_karma_webpack_/webpack:/src/app/modules/adjudication/pages/enrollee-adjudicator-documents/enrollee-adjudicator-documents.component.ts:56:4)
      at SafeSubscriber.__tryOrUnsub (http://localhost:9876/_karma_webpack_/webpack:/node_modules/rxjs/_esm2015/internal/Subscriber.js:183:1)
      at SafeSubscriber.next (http://localhost:9876/_karma_webpack_/webpack:/node_modules/rxjs/_esm2015/internal/Subscriber.js:122:1)
      at Subscriber._next (http://localhost:9876/_karma_webpack_/webpack:/node_modules/rxjs/_esm2015/internal/Subscriber.js:72:1)
      at Subscriber.next (http://localhost:9876/_karma_webpack_/webpack:/node_modules/rxjs/_esm2015/internal/Subscriber.js:49:1)
      at Object.complete (http://localhost:9876/_karma_webpack_/webpack:/node_modules/rxjs/_esm2015/internal/observable/forkJoin.js:50:1)
      at Object.wrappedComplete (http://localhost:9876/_karma_webpack_/webpack:/node_modules/rxjs/_esm2015/internal/Subscriber.js:166:1)
      at SafeSubscriber.__tryOrUnsub (http://localhost:9876/_karma_webpack_/webpack:/node_modules/rxjs/_esm2015/internal/Subscriber.js:183:1)
      at SafeSubscriber.complete (http://localhost:9876/_karma_webpack_/webpack:/node_modules/rxjs/_esm2015/internal/Subscriber.js:168:1)
      at Subscriber._complete (http://localhost:9876/_karma_webpack_/webpack:/node_modules/rxjs/_esm2015/internal/Subscriber.js:79:1)
Chrome Headless 98.0.4758.102 (Linux x86_64) ERROR
  An error was thrown in afterAll
  TypeError: Cannot read properties of undefined (reading 'removeFiles')
      at SafeSubscriber._next (http://localhost:9876/_karma_webpack_/webpack:/src/app/modules/adjudication/pages/enrollee-adjudicator-documents/enrollee-adjudicator-documents.component.ts:56:4)
      at SafeSubscriber.__tryOrUnsub (http://localhost:9876/_karma_webpack_/webpack:/node_modules/rxjs/_esm2015/internal/Subscriber.js:183:1)
      at SafeSubscriber.next (http://localhost:9876/_karma_webpack_/webpack:/node_modules/rxjs/_esm2015/internal/Subscriber.js:122:1)
      at Subscriber._next (http://localhost:9876/_karma_webpack_/webpack:/node_modules/rxjs/_esm2015/internal/Subscriber.js:72:1)
      at Subscriber.next (http://localhost:9876/_karma_webpack_/webpack:/node_modules/rxjs/_esm2015/internal/Subscriber.js:49:1)
      at Object.complete (http://localhost:9876/_karma_webpack_/webpack:/node_modules/rxjs/_esm2015/internal/observable/forkJoin.js:50:1)
      at Object.wrappedComplete (http://localhost:9876/_karma_webpack_/webpack:/node_modules/rxjs/_esm2015/internal/Subscriber.js:166:1)
      at SafeSubscriber.__tryOrUnsub (http://localhost:9876/_karma_webpack_/webpack:/node_modules/rxjs/_esm2015/internal/Subscriber.js:183:1)
      at SafeSubscriber.complete (http://localhost:9876/_karma_webpack_/webpack:/node_modules/rxjs/_esm2015/internal/Subscriber.js:168:1)
      at Subscriber._complete (http://localhost:9876/_karma_webpack_/webpack:/node_modules/rxjs/_esm2015/internal/Subscriber.js:79:1)
Chrome Headless 98.0.4758.102 (Linux x86_64): Executed 543 of 605 ERROR (11.556 secs / 6.288 secs)
```

Reply:
Hmmm, I thought I already fixed that, will look into it again.